feature-libs/customer-ticketing/root/model/augmented-core.model.ts

```diff
@@ -2,6 +2,7 @@ import '@spartacus/storefront';
 
 declare module '@spartacus/storefront' {
   const enum LAUNCH_CALLER {
+    CUSTOMER_TICKETING_CREATE = 'CUSTOMER_TICKETING_CREATE',
```

Comment:
👍 

line 6 is something I missed that should be handled in the refactor of Parth's reopen/close PR